web/components/IssueForm/IssueUrlField/IssueUrlField.js

```diff
@@ -0,0 +1,85 @@
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+import { FieldGroup } from 'components/shared/FieldGroup';
+import { FormField } from 'components/shared/FormField';
+
+import { useUniqueURLValidation } from './useUniqueURLValidation';
+
+export function IssueUrlField({
+  value,
+  error: validationError,
+  touched,
+  onBlur,
+  onChange,
+}) {
+  const [
+    isUniqueURL,
+    isValidatingURL,
+    errorValidatingURL,
+  ] = useUniqueURLValidation(value);
+
+  const error = getError({
+    isUniqueURL,
+    isValidatingURL,
+    errorValidatingURL,
+    validationError,
+    touched,
+  });
+
+  return (
+    <FieldGroup
+      renderTitle={(className) => (
+        <label htmlFor="url-input" className={className}>
+          Issue Url
+        </label>
+      )}
+    >
+      <FormField error={error}>
+        <input
+          type="text"
+          name="issueUrl"
+          id="url-input"
+          className={classnames('border border-gray-500 rounded-none w-full', {
```

Comment:
i think class "bordedr-gray-500" should be with condition !error on error === undefined, so if there is an error the 2 classes will added and the last one will override the first one.
What do you think?

Reply:
you're 100% correct. fixed